### DISCUSSION TITLE
Adapting Monash FIT3143 - Parallel Computing unit content for training book

### DISCUSSION THREAD

**Discussion by oraqlle** [MonashDeepNeuron/HPC-Training#9] [Ideas]
@UnciaBit had the idea of adapting the content from Monash's Parallel Computing unit [FIT3143](https://handbook.monash.edu/current/units/FIT3143) for our purposes in the HPC training book.

@mmib0001 Is there anything against this in the MDN policy and what are your thoughts on this?

**Comment by mmib0001** [comment]
I think all of the previous trainings had some relationship with Monash content, I think it's fine if it is for internal purposes only @oraqlle

**Reply by oraqlle** [reply]
This book and repo is public though if that poses an issue.
We'll most likely just strip the ideas out for our own purposes.

**Reply by mmib0001** [reply]
I'm checking with exec rn, I think you can use the ideas but not the specific content, I'll confirm when I can

**Reply by mmib0001** [reply]
@oraqlle **Confirmed** adaption is fine, just not fully copying

**Comment by oraqlle** [comment]
@UnciaBit you are free to utilize FIT3143 content. I can send an email to the lecturer or chief examiner if you need.

**Reply by isoyuki** [reply]
Great. Would you be able to shoot an email to the chief examiner to ask if they can provide with an outline of the topics covered and/or other materials they're willing to provide?
